asyncpgsa/pool.py

```diff
@@ -5,10 +5,16 @@
 
 
 class SAPool(Pool):
+    __slots__ = ('_debug',)
```

Comment:
I had to validate myself too, seemed strange - but fitting of the data model since it likely simplifies the inheritance chain for the core.